src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -31,10 +31,13 @@
 from typing import Union
 from typing import overload
 from typing import Literal
+from typing import Any
```

Comment:
Could you move this all to a new module. Define this as a class called `mvd_info` or sth. And make that accessible from `ifcopenshell.file.mvd. ...`. That way it's a bit grouped. Bonus points for making this degrade nicely when lark is not installed.